app/models/user.rb

```diff
@@ -2,7 +2,7 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Kudo uncommenting `:confirmable`. 👍️
And please make sure that it's working too after fixing the issue with the registration.